axelor-base/src/main/resources/views/Company.xml

```diff
@@ -57,6 +57,9 @@
 	    	<panel name="informationsPanel" title="Information">
 	      		<field name="notes" colSpan="12"/>
 	    		<field name="companyDepartmentList" grid-view="company-department-company-grid" form-view="company-department-company-form" canSelect="true" domain="self.company IS NULL" colSpan="12"/>
+    			<field name="partnerList" form-view="partner-form" grid-view="partner-grid" colSpan="12" canNew="false" canSelect="true" onSelect="action-company-partner-list-on-select"/>
+		    	<field name="workshopList" colSpan="12" if-module="axelor-stock" if="__config__.app.isApp('stock')" canNew="false" canSelect="true" onSelect="action-company-workshop-list-on-select"/>
```

Comment:
I think this field should only appears if the workshop configuration is enabled.